### PR TITLE
Ocd 2351 - nonconformity charts

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 ## Version 14.4.1
 _Date TBD_
 
+### Table Addition
+* Add new nonconformity statistics table
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]: Should be in PENDING_PENDING_RELEASE_NOTES as STG is frozen

File: criteria.csv [hunk 2]
```diff
@@ -0,0 +1,133 @@
+deleted,number,value
```
Comment (andlar) [inline criteria.csv]: What's this file for?

File: dev/update_users.sh [hunk 3]
```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```
Comment (andlar) [inline dev/update_users.sh]: All of these users files shouldn't be here. The "maint/users.sql" file is ignored on purpose

File: v-next.sql [hunk 4]
```diff
@@ -45,6 +45,22 @@ where openchpl.can_add_new_status(db_id, eff_date, chpl_id) = true;
     end;
     $$ language plpgsql;
 
+-- OCD - 2351 - nonconformity chart statistics
```
Comment (andlar) [inline v-next.sql]: Should be in v-dev.sql

File: v14.4.1.sql [hunk 5]
```diff
@@ -45,6 +45,22 @@ where openchpl.can_add_new_status(db_id, eff_date, chpl_id) = true;
     end;
     $$ language plpgsql;
 
+-- OCD - 2351 - nonconformity chart statistics
```
Comment (kekey1) [inline v14.4.1.sql]: move to v-dev

File: v-dev.sql [hunk 6]
```diff
@@ -0,0 +1,16 @@
+-- OCD - 2351 - nonconformity chart statistics
+
+DROP TABLE openchpl.nonconformity_type_statistics;
```
Comment (andlar) [inline v-dev.sql]: Should this be "DROP TABLE IF EXISTS openchpl....."?

Also, needs to be removed from the v14.something file

File: v14.4.1.sql [hunk 7]
```diff
@@ -45,6 +45,8 @@ where openchpl.can_add_new_status(db_id, eff_date, chpl_id) = true;
     end;
     $$ language plpgsql;
 
+CREATE TRIGGER nonconformity_type_statistics_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.nonconformity_type_statistics FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
```
Comment (andlar) [inline v14.4.1.sql]: Still needs to be removed

**Pull request merged**